zetaclient/ethrpc/types.go

```diff
@@ -0,0 +1,162 @@
+package ethrpc
+
+import (
+	"bytes"
+	"math/big"
+	"unsafe"
+)
+
```

Comment:
Do we need to maintain these types or can they be shared with the other rpc client?

Reply:
let me make a comment in borrow file
https://github.com/onrik/ethrpc/blob/main/types.go

Reply:
the [commit](https://github.com/zeta-chain/node/pull/1665/commits/003e851e10e5f76fee146ac76cc4bb564b6c523b)